modules/persistent-storage-vsphere-backup.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-pv-backup_{context}"]
+= Backing up VMware vSphere volumes
+
+{product-title} provisions new volumes as independent persistent disks to freely attach and detach the volume on any node in the cluster. As a consequence, it is not possible to back up volumes that use snapshots, or restore volumes from created snapshots. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[Snapshot Limitations] for more information.
+
+.Procedure
+
+To create a backup of persistent volumes:
+
+.  Stop the application that is using the persistent volume.
+.  Clone the persistent disk.
```

Comment:
For consistency and clarity, instead of "disk", should we stick to "volume" in all these steps? I'm not clear as to actual manifestation of how OCP provisions volumes as disks. If the terminology is interchangeable in this setting, then better to use one or the other in the procedure. But if it's not, then fine as it is. 😀

Reply:
> For consistency and clarity, instead of "disk", should we stick to "volume" in all these steps? I'm not clear as to actual manifestation of how OCP provisions volumes as disks. If the terminology is interchangeable in this setting, then better to use one or the other in the procedure. But if it's not, then fine as it is. 😀

@huffmanca, Can you please confirm that "disk" and "volume are interchangeable throughout this procedure? Thank you.